src/Nova/NovaMailEvent.php

```diff
@@ -8,6 +8,7 @@
 use Laravel\Nova\Fields\Text;
 use Laravel\Nova\Fields\BelongsTo;
 use KirschbaumDevelopment\NovaMail\Models\NovaMailEvent as NovaMailEventModel;
+use Laravel\Nova\Http\Requests\NovaRequest;
```

Comment:
Not sure if we are using PHPCS Fixer, but this line should be moved up one for import length

Reply:
Good call, I think my config wasn't working correctly for some reason.